eoscompanion/routes.py

```diff
@@ -63,6 +62,8 @@
 from .ekn_query import ascertain_application_sets_from_models
 from .format import (
     format_app_icon_uri,
+    format_thumbnail_uri,
```

Comment:
It does not look like this import is needed in this commit

Reply:
Ah, you mean `format_thumbnail_uri` ?

Reply:
Hmm, both `format_content_data_uri`, `format_thumbnail_uri` and `optional_format_thumbnail_uri` all seem to be used below. Not sure what you're referring to here?